src/components/callback-notifications.tsx

```diff
@@ -29,11 +29,6 @@ export const CallbackNotifications = ({}) => {
           info: <>New metadata:</>,
           details: JSON.stringify(event.data.metadata, null, 2),
         },
-        "update-file": {
-          title: "Update file contents",
-          info: <>New file contents:</>,
-          details: JSON.stringify(event.data.content, null, 2),
-        },
```

Comment:
this still runs, right? but way more often, so this is probably super annoying

Reply:
yeah, exactly, `onRequestUpdateContent` runs on (e.g.) every keystroke in the CodeMirror block.